installers/local/carbon.cpp

```diff
@@ -1,7 +1,3 @@
-// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
-// Exceptions. See /LICENSE for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
```

Comment:
It looks like you accidentally removed the copyright header here

Reply:
Made that a little bit messy, thanks for the link! I'll read it and then start to commit :))